### ISSUE TITLE
F-Droid build is buggy on real devices

### ISSUE THREAD

**Issue by NobodyForNothing** [NobodyForNothing/blood-pressure-monitor-fl#345]
When freshly installing the f-droid build on a real device the add measurement button doesn't work. This means the following code fails:
```dart
final recordRepo = RepositoryProvider.of<BloodPressureRepository>(this);
final noteRepo = RepositoryProvider.of<NoteRepository>(this);
final intakeRepo = RepositoryProvider.of<MedicineIntakeRepository>(this);
final exportSettings = Provider.of<ExportSettings>(this, listen: false);

final entry = await showAddEntryDialoge(this,
  RepositoryProvider.of<MedicineRepository>(this),
  initial,
);

---

Future<FullEntry?> showAddEntryDialoge(
  BuildContext context,
  MedicineRepository medRepo,
  [FullEntry? initialRecord,
]) async {
  final meds = await medRepo.getAll();
  return showDialog<FullEntry>(
    context: context, builder: (context) => AddEntryDialoge(
      initialRecord: initialRecord,
      availableMeds: meds,
    ),
  );
}
```

**Comment by NobodyForNothing** [comment]
Error log:
```
Error:
Build number:35
-----
Error opening add measurement dialoge:
---
DatabaseException(no such column: false (code 1 SQLITE_ERROR): , while compiling: SELECT designation, defaultDose, color FROM Medicine WHERE removed = false) sql 'SELECT designation, defaultDose, color FROM Medicine WHERE removed = false' args []
#0      wrapDatabaseException (package:sqflite/src/exception_impl.dart:11:7)
<asynchronous suspension>
#1      SqfliteDatabaseMixin.txnRawQuery.<anonymous closure> (package:sqflite_common/src/database_mixin.dart:586:30)
<asynchronous suspension>
#2      BasicLock.synchronized (package:synchronized/src/basic_lock.dart:33:16)
<asynchronous suspension>
#3      SqfliteDatabaseMixin.txnSynchronized (package:sqflite_common/src/database_mixin.dart:517:14)
<asynchronous suspension>
#4      MedicineRepositoryImpl.getAll (package:health_data_store/src/repositories/medicine_repository_impl.dart:37:21)
<asynchronous suspension>
#5      showAddEntryDialoge (package:blood_pressure_app/features/input/add_measurement_dialoge.dart:425:16)
<asynchronous suspension>
#6      EntryUtils.createEntry (package:blood_pressure_app/data_util/entry_context.dart:26:21)
<asynchronous suspension>

```

**Comment by NobodyForNothing** [comment]
F-Droid build recipe:
```yaml
- versionName: 1.7.0
    versionCode: 35
    disable: https://github.com/NobodyForNothing/blood-pressure-monitor-fl/issues/340
    commit: d407082821df2f90848c4e87dd8ede0eea16497b
    subdir: app
    output: build/app/outputs/flutter-apk/app-fdroid-release.apk
    srclibs:
      - flutter@beta
    rm:
      - app/ios
      - app/linux
      - app/macos
      - app/web
      - app/windows
      - app/test
      - docs
    prebuild:
      - export flutterVersion=$(sed -n -E "s/.*f-droid-flutter:\s*(.*)/\1/p" pubspec.yaml)
      - '[[ $flutterVersion ]] || exit 1'
      - pushd $$flutter$$
      - git checkout -f $flutterVersion
      - popd
      - export PUB_CACHE=.pub-cache
      - $$flutter$$/bin/flutter config --no-analytics
      - $$flutter$$/bin/flutter packages pub get
    scandelete:
      - app/.pub-cache
    build:
      - export PUB_CACHE=.pub-cache
      - $$flutter$$/bin/flutter build apk --flavor fdroid --release --obfuscate --split-debug-info=./build/debug-info
```

seems to build health_data_store incorrectly

**Comment by NobodyForNothing** [comment]
Potential issues:
- PUB_CACHE might needs to be the same for package and builds
- `dart run build_runner build` might needs to be in the build section

**Comment by NobodyForNothing** [comment]
fixed in 0a01339
